### ISSUE TITLE
Cloud Environment: Icons not updating while solution is loading.

### ISSUE THREAD

**Issue by jjmew** [dotnet/project-system#5900]
**Visual Studio Version**: Version 16.6.0 Preview 1.0 [29817.27.master] 

**Summary**:
When building the project-system repo you see warning icons on the dependencies node

**Steps to Reproduce**:

1. Create an Online Environment pointing to the project system rep

2. Build the solution

**Expected Behavior**:
No warning icons
**Actual Behavior**: 

![image](https://user-images.githubusercontent.com/14163814/74779620-b1ae2f80-5252-11ea-9585-cb5444818af6.png)

**User Impact**:

**Comment by melytc** [comment]
@drewnoakes and I tested this, and we noticed that the icons load depending on the moment in time you expand the node.

If you expand it **right away** when you opened the solution:
![image](https://user-images.githubusercontent.com/8518253/75828701-5192af80-5d61-11ea-953f-91013afaad4f.png)

But if you **wait** until the whole solution is loaded without opening any node in the solution (around 5 min., but could take less than that), you get the cleaned/expected behavior:
![image](https://user-images.githubusercontent.com/8518253/75828748-696a3380-5d61-11ea-9179-091e21a751a3.png)

We found out that you would have all sorts of combinations in between opening the solution and waiting until it's completed loaded: some with warnings in the ProjectSystemSetup but not in the actual project, for example.

This makes us thing that it might not our issue, @jjmew. I'm not sure whose is it.
